src/components/Game.java

```diff
@@ -59,32 +59,30 @@ public void parse() {
                     startScene.parse();
                     break;
                 case "STORY SCENES":
-                    while(!Main.literals.contains(tokenizer.checkNext())) {
+                    while(!Main.literals.contains(tokenizer.checkNext()) && !tokenizer.checkNext().equals("NO_MORE_TOKENS")) {
```

Comment:
If there are no more tokens, checkNext actually returns a token of NO_MORE_TOKENS. Assuming we can write story, death, and end scenes out of order, we gotta check for this token to ensure we don't loop forever